plugins/blog/mod.ts

```diff
@@ -3,9 +3,9 @@ import type { Plugin } from "$fresh/server.ts";
 import BlogIndex from "./routes/blog/index.tsx";
 import BlogSlug from "./routes/blog/[slug].tsx";
 import Feed from "./routes/feed.ts";
-import { normalize } from "$std/url/normalize.ts";
+import { normalize } from "@std/url";
 
-export function blog(): Plugin {
+export function blog(): Plugin & { location: string; projectLocation: string } {
```

Comment:
might need some help on this. I couldn't seem to find any documentation about the `location` and `projectLocation` options. So I am not sure what to do here.